starknet_py/tests/e2e/docs/account_creation/test_deploy_prefunded_account.py

```diff
@@ -43,26 +49,30 @@ async def test_deploy_prefunded_account(
     await res.wait_for_acceptance()
     # docs: start
 
-    # Create an Account instance
-    account = Account(
-        address=address,
-        client=GatewayClient(net=network),
-        key_pair=key_pair,
-        chain=StarknetChainId.TESTNET,
-    )
+    # Define the client to be used to interact with StarkNet
+    client = GatewayClient(net=TESTNET)
+    chain = StarknetChainId.TESTNET
+    # docs: end
+
+    client = gateway_client
+    chain = chain_from_network(net=network, chain=StarknetChainId.TESTNET)
```

Comment:
I'm having second thoughts about using this function, as #719 is a thing now, and also I don't really like the way it works (and the fact that it's chain from network but it still takes chain). Maybe let's not include it in docs, my bad 😶

Reply:
It is not in the docs ;)